test/acceptance/tests.py

```diff
@@ -75,6 +75,9 @@ class OpenAssessmentTest(WebAppTest):
         'full_workflow_staff_required':
             u'courses/{test_course_id}/courseware/'
             u'8d9584d242b44343bc270ea5ef04ab03/0b0dcc728abe45138c650732af178afb/'.format(test_course_id=TEST_COURSE_ID),
+        'feedback_only':
```

Comment:
Yeah, but by the same token, the primary purpose of this test is to check a criterion with only feedback (no scoring options). @dianakhuang, can you think of a better name?

I also call it the "Feedback Only Subsection" in the LMS, I'll need to change that as well.